platform_plugin_aspects/utils.py

```diff
@@ -121,6 +107,12 @@ def _generate_guest_token(user, course, superset_config, dashboards, filters):
     }
 
     try:
+        client = SupersetClient(
+            host=superset_internal_host,
+            username=superset_username,
+            password=superset_password,
+        )
+
         logger.info(f"Requesting guest token from Superset, {data}")
```

Comment:
I think `debug` would be a log level more appropriate for this. It was added for debugging and I missed to remove it